<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCameraControl</class>
 <widget class="QDialog" name="DialogCameraControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>摄像头控制定点</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btConnect">
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>服务器:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leServer">
      <property name="text">
       <string>localhost</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>端口:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lePort">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>4892</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>271</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>状态</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>当前位置:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leCurrentPosition">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>目标位置:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leTargetPosition">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>偏差(直):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leDeviationXy">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>偏差(极):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leDeviationPole">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>参数</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>P:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leP">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>I:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leI">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>D:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leD">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>系数:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leScale">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>单位：度/像素</string>
       </property>
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>限幅:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leClipping">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>单位：度</string>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>49</y>
     <width>101</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="btTargetAssignCurrent">
      <property name="text">
       <string>设当前为目标</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btLockUnlockParam">
      <property name="text">
       <string>解锁参数</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btConnect</tabstop>
  <tabstop>leServer</tabstop>
  <tabstop>lePort</tabstop>
  <tabstop>btTargetAssignCurrent</tabstop>
  <tabstop>btLockUnlockParam</tabstop>
  <tabstop>leP</tabstop>
  <tabstop>leI</tabstop>
  <tabstop>leD</tabstop>
  <tabstop>leScale</tabstop>
  <tabstop>leClipping</tabstop>
  <tabstop>leCurrentPosition</tabstop>
  <tabstop>leTargetPosition</tabstop>
  <tabstop>leDeviationXy</tabstop>
  <tabstop>leDeviationPole</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
